content/about/about_people.rst

```diff
@@ -201,3 +201,38 @@ Vancelot
 .. raw:: html
 
   <br/>
+
+`Andrew Dassonville`_
+----------------------
+
+.. image:: /images/andrewda.jpg
+    :width: 110px
+    :align: right
+    :alt: Andrew Dassonville
+
+.. class:: no-breaks
+
+  **Student Systems Engineer**
+
+Andrew joined the OSL in October 2019 and is studying computer science. He first
+learned about the OSL during the College of Engineering's "Clubs and Cookies"
+day, and immediately applied for a position. Andrew joined the lab because he is
```

Comment:
```suggestion
day and immediately applied for a position. Andrew joined the lab because he is
```
Unnecessary comma in a [compound predicate](https://www.grammar.com/compound-predicate)